### PR TITLE
Added ability to handle command line args in scenarios

### PR DESCRIPTION
**Pull request by dutow**
Also moved the configuration logic to rely on command line arguments instead of environment variables, both for the configuration file customization and specifying the postgres installation location.

This should be simpler to use than the previous environment variable approach, but environment variables with getenv are still available to scenarios.

### CODE REVIEW

File: scripts/argparser.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
+local argparse = require("argparse")
+
+argparser = argparse("stormweaver", "")
+argparser:argument("scenario", "Scenario file")
+argparser:option("-c --config", "Configuration file.", "config/stormweaver.toml")
+argparser:option(
+	"-i --install_dir",
+	"PostgreSQL installation directory (if specified, overrides configuration value for default server)",
+	""
+)
```
Comment (dAdAbird) [inline scripts/argparser.lua:10]: Not a blocker, just wondering, does it makes sense to have an option for datadir_root as well?

**Review by dAdAbird — APPROVED** [review]

File: scripts/argparser.lua (hunk 1 above)
Comment (dutow) [reply scripts/argparser.lua:10]: We might add something like that in the future, but it is less critical than the installation path. It's a corner case if you want to run the tests on a different disk or something.
